<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCropWidget</class>
 <widget class="QWidget" name="EditCropWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Plot:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ComboBoxKey" name="plotInput">
       <property name="toolTip">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PhysInput" name="shapeInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="EditPlotsBtn">
       <property name="text">
        <string>Edit plots</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Plant:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ComboBoxKey" name="plantInput">
       <property name="toolTip">
        <string>Plant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ComboBoxKey" name="varInput">
       <property name="toolTip">
        <string>Variety</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditPlantsBtn">
       <property name="text">
        <string>Edit plants</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Planned:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="EnableCalendar" name="plannedstartdateInput">
       <property name="toolTip">
        <string>Planned start date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="EnableCalendar" name="plannedenddateInput">
       <property name="toolTip">
        <string>Planned end date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="EnableCalendar" name="startdateInput">
       <property name="toolTip">
        <string>Start date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2e">
       <property name="text">
        <string>Actual:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="EnableCalendar" name="enddateInput">
       <property name="toolTip">
        <string>End date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>End</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="noteInput">
     <property name="toolTip">
      <string>Note</string>
     </property>
     <property name="placeholderText">
      <string>Add note here</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QPushButton" name="AddButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add this crop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnableCalendar</class>
   <extends>QDateEdit</extends>
   <header>EnableCalendar.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxKey</class>
   <extends>QComboBox</extends>
   <header>ComboBoxKey.h</header>
  </customwidget>
  <customwidget>
   <class>PhysInput</class>
   <extends>QLineEdit</extends>
   <header>PhysInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
